Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM golang:1.19.9-alpine3.16 as builder
+
+RUN apk add --no-cache make gcc musl-dev linux-headers jq bash
+
+WORKDIR /app
+
+COPY . .
+RUN make build-app
+
+FROM alpine:3.16
+WORKDIR /app
+COPY --from=builder /app/bin/pessimism .
+COPY config.env .
+COPY genesis.example.json .
+CMD ["./pessimism"]
```

Comment:
Can we add some comments on how we're performing a multi-image build and the rationale behind?